src/App.jsx

```diff
@@ -3,26 +3,86 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getTodos, getUserInfo } from './api/api';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    title: '',
+    isCompleted: '',
```

Comment:
move this values to the TodoList component state, entire App shouldn't know about filtering values